tests/cpp/test_compiled_modules.cpp

```diff
@@ -1,65 +0,0 @@
-#include "cpp_api_test.h"
```

Comment:
We should leave these tests with a significantly reduced set of models to verify the C++ API end to end

Reply:
Yeah I scaled down the number of models to 5. This test is removed in C++ and added in python. Check out tests/models/test_models.py which is equivalent of this with reduced set of models and compares with pytorch nn.module directly instead of torchscript module with cosine sim

Reply:
Well we still need to verify the C++ apis so the python tests are not 100% equivalent

Reply:
So a question - In that case, we will compare torch.jit.script result with TRT engine result which is okay ? The reason I ask is we saw even torch.jit.script results differed from nn.Module in terms of correctness (not cosine sim) so I thought a proper comparison will only be with nn.Module

Reply:
Maybe one thing we could do is have the test in c++ and not compare the end result so to ensure functionality wise C++ API works ?

Reply:
We could just have a very generous tolerance

Reply:
added it back